CMakeLists.txt

```diff
@@ -164,6 +165,32 @@ if(NOT mxnet_LINKER_LIBS)
   set(mxnet_LINKER_LIBS "")
 endif(NOT mxnet_LINKER_LIBS)
 
+if(USE_NGRAPH AND NOT MSVC)
+  include(cmake/ngraph.cmake)
+  if(NOT MSVC)
```

Comment:
`NOT MSVC` is already guaranteed a few lines up.  Is this logic really what you intended?